lnwallet/wallet.go

```diff
@@ -39,6 +39,13 @@ const (
 	// TODO(halseth): update constant to target a specific commit size at
 	// set fee rate.
 	anchorChanReservedValue = btcutil.Amount(10_000)
+
+	// maxAnchorChanReservedValue is the maximum value we'll reserve for
+	// anchor channel fee bumping. We cap it at 10 times the per-channel
+	// amount such that nodes with a high number of channels don't have to
+	// keep around a very large amount for the unlikely scanario that they
```

Comment:
It is indeed somewhat arbitrary, as all we can do is make an educated guess w.r.t how the amount of fees we'll actually need at _close time_. Which shifts around the problem from the prior instance where we needed to know (and over guess) at any point that the remote peer is actually online. 

After my ML post on the lnd mailing list, I realized that the existing settings would cause wallet operators like Breez to  need to have a ton of sats sitting around as they potentially have thousands of private channels to thier node.

Reply:
It's also the case that _all_ channels closing at the same time is an unlikely event in practice, and if we can re-use the UTXO to bump for all of them, then we require less spare outputs on chain.